### DISCUSSION TITLE
change index for custom

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#6987] [Help]
How can I change the index of a resource so that it points to a custom page from the main menu of the application?

**Comment by danharrin** [comment]
What are you trying to do

**Reply by jcgmalave** [reply]
for a specific user role, I need to directly show one of the records that is already loaded, for example this route: Pages\EditWarehouseconfig::route('/1/edit') and for another user role the conventional index : Pages\ListWarehouseconfigs::route('/'). Is it possible?

**Reply by danharrin** [reply]
Probably easier to just create a separate custom page and then share the form schema from the resource?
